contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -328,6 +328,18 @@ public void testOpenProcess() {
     	assertEquals(WinError.ERROR_ACCESS_DENIED, Kernel32.INSTANCE.GetLastError());
     }
 
+    public void testQueryFullProcessImageName() {
+    	HANDLE h = Kernel32.INSTANCE.OpenProcess(0, false,
+    			Kernel32.INSTANCE.GetCurrentProcessId());
+    	assertNotNull("Failed (" + Kernel32.INSTANCE.GetLastError() + ") to get process handle", h);
+    	
+    	char[] path = new char[WinDef.MAX_PATH];
+        IntByReference lpdwSize = new IntByReference(path.length);
```

Comment:
Just add a line in _Changes.MD_ under "Next release" features - same as the the other lines.